### PR TITLE
Standardize logging with LOGNAME variable

### PR DESCRIPTION
**Pull request by cambel**
### Description

Updating the logging name variable `name_` to `LOGNAME` to keep consistency with other modules.

### Checklist
- [x] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)

### CODE REVIEW

**Review by simonschmeisser — APPROVED** [review]

File: moveit_ros/planning/trajectory_execution_manager/include/moveit/trajectory_execution_manager/trajectory_execution_manager.h [hunk 1]
```diff
@@ -301,7 +301,7 @@ class TrajectoryExecutionManager
   void loadControllerParams();
 
   // Name of this class for logging
-  const std::string name_ = "trajectory_execution_manager";
+  const std::string LOGNAME = "trajectory_execution_manager";
```
Comment (rhaschke) [inline moveit_ros/planning/trajectory_execution_manager/include/moveit/trajectory_execution_manager/trajectory_execution_manager.h:304]: An upper-case variable indicates a global or static one. Thus, this should be changed to `static` as well:
```suggestion
  static const std::string LOGNAME = "trajectory_execution_manager";
```

**Review by rhaschke — COMMENTED** [review]
Minor nitpick. Otherwise looks good.

File: moveit_ros/planning/trajectory_execution_manager/include/moveit/trajectory_execution_manager/trajectory_execution_manager.h (hunk 1 above)
Comment (cambel) [reply moveit_ros/planning/trajectory_execution_manager/include/moveit/trajectory_execution_manager/trajectory_execution_manager.h:304]: sure

File: moveit_ros/planning/trajectory_execution_manager/include/moveit/trajectory_execution_manager/trajectory_execution_manager.h [hunk 2]
```diff
@@ -301,7 +301,7 @@ class TrajectoryExecutionManager
   void loadControllerParams();
 
   // Name of this class for logging
-  const std::string name_ = "trajectory_execution_manager";
+  static const std::string LOGNAME = "trajectory_execution_manager";
```
Comment (simonschmeisser) [inline moveit_ros/planning/trajectory_execution_manager/include/moveit/trajectory_execution_manager/trajectory_execution_manager.h:304]: Getting closer :laughing: Other classes define this as a non-member in the cpp file

Comment (cambel) [reply moveit_ros/planning/trajectory_execution_manager/include/moveit/trajectory_execution_manager/trajectory_execution_manager.h]: Okay

**Pull request merged**